include/swift/AST/Attr.def

```diff
@@ -359,6 +359,10 @@ SIMPLE_DECL_ATTR(rethrows, Rethrows,
   RejectByParser |
   ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
   57)
+SIMPLE_DECL_ATTR(rethrows, AtRethrows,
+  OnProtocol |
+  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
```

Comment:
I don't think it's API-stable to remove, is it?

Reply:
hmm good point, yea this is not API stable to remove.

Reply:
hmm so the previous annotation btw marks regular `rethrows` API stable to remove, isn't that wrong?